bundles/org.eclipse.swt.svg/META-INF/MANIFEST.MF

```diff
@@ -11,3 +11,6 @@ Import-Package: com.github.weisj.jsvg;version="[1.7.0,2.0.0)",
  com.github.weisj.jsvg.geometry.size;version="[1.7.0,2.0.0)",
  com.github.weisj.jsvg.parser;version="[1.7.0,2.0.0)"
 Export-Package: org.eclipse.swt.svg
+Provide-Capability: osgi.implementation;
+    osgi.implementation="swt.svgrasterizer";
+    version:Version="1.0"
```

Comment:
With the given reference I have the impression that the `osgi.implementation` namespace is inteded for implementations of OSGi specifications not arbitrary specs.
While it would probably not be harmful to use the osgi-namespace, I think a custom one would be more suitable, for example:
```suggestion
Provide-Capability: eclipse.swt.implementation;image.format="svg";version:Version="1.0"
```
Later we could then also use this namespace to introduce a requirement of o.e.swt to it's name fragments that could then provide `Provide-Capability: eclipse.swt.implementation;implementation="native"` or similar.